content/tracking/tracking_1.md

```diff
@@ -0,0 +1,111 @@
++++
+title = "Setup your tracking"
+weight = 1
++++
+
+Depending on your frontend infrastructure and Snowplow setup, you can use either our JavaScript tracker for a `<script>` tag option or our Browser tracker for a more modern web development setup.
+
+In both options, the API is similar with only minor differences in the setup and method calls.
+
+{{< tabs groupId="package manager" >}}
+{{% tab name="Package Manager" %}}
+
+#### **Step 1:** Install browser-tracker package
+
+Install the `@snowplow/browser-tracker` and `@snowplow/browser-plugin-enhanced-consent` via npm, yarn or any other package manager of your choice. Example using `npm`:
+
+```bash
+npm install @snowplow/browser-tracker @snowplow/browser-plugin-enhanced-consent
+```
+
+---
+
+#### **Step 2:** Create the tracker
+
+In your `src` folder, create a file called `tracker.js`. Inside it create the `tracker` object using the snippet below to use it anywhere in the application:
+
+- Tracker Name: `'sp'`
+- Collector Url: `'{{Url for Collector}}'`
+
+```javascript
+import { newTracker } from "@snowplow/browser-tracker";
+
+export const tracker = newTracker("sp", "{{Url for Collector}}", {
+  /* tracker options */
+});
+```
+
+In addition to the basic tracker, you can add any number of options for using Snowplow's more advanced features. <a target="_blank" href="https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/tracker-setup/initialization-options/">See more on our documentation</a>.
+
+---
+
+#### **Step 3:** Configure the tracker to use the `EnhancedConsentPlugin`
```

Comment:
```suggestion

```
As the web accelerator is a pre-req, lets merge these steps. We could also include the same recommended settings we used there for continuity

Reply:
On that front, what I think would be a good idea is to make this an expandable section which on the advanced accelerators is by default closed, but I would not remove it completely.

WDYT ?

Reply:
Sorry am a bit lost here. Is this in regards to step2 or step3?

Reply:
We have added the Advanced Analytics accelerator as a 'prerequisite' so probably the reader has gone through these steps previously.

My suggestion would be not to remove the instructions completely but just to create an expansion panel with steps 1, 2 and 3 which would be closed by default.

**Note**
I would not consider this a blocker for now.

Reply:
Think an expansion panel would work well here.